README.md

```diff
@@ -39,3 +39,79 @@ Every query returns a JSON object with a `TYPE` attribute which is either `cours
 
 After you get an app working with the dummy data from `get_info.php`, you can tackle getting a mariadb instance
 set up and having a php script gather information from a database.
+
+## Database Setup
+Login to database via Mariadb (I used MYSQL client via windows)
+
+Once logged, do the following to create your local database and user to allow access via php:
+### 1. Database creation
+```
+MariaDB[(none)]> CREATE DATABASE `mydb`;
+```
+### 2. User creation
+```
+MariaDB[(none)]> CREATE USER 'myuser' IDENTIFIED BY 'mypassword';
+```
+### 3. Grant permissions to access and use the MySQL server
+Only allow access from localhost (this is the most secure and common configuration you will use for a web application):
+```
+MariaDB[(none)]> GRANT USAGE ON *.* TO 'myuser'@localhost IDENTIFIED BY 'mypassword';
+```
+To allow access to MySQL server from any other computer on the network:
```

Comment:
I don't think we need this. Did you have a usecase in mind?

Reply:
I was thinking that although we test on our local servers, it might be a good idea to have one database with a very detailed set of test data to operate on(via a online web server). With that being said, we also could continue to update one sql file with more coverage on test cases as we go.

Reply:
I think the security weaknesses aren't worth it. There's nothing wrong with a big `.sql` file that creates plenty of test data.